packages/dlp/test/gapic_v2beta1.js

```diff
@@ -0,0 +1,341 @@
+/*
```

Comment:
That's what we configured for video intelligence. @lukesneeringer Can you double check?